etcdctl/ctlv2/command/util.go

```diff
@@ -203,7 +203,7 @@ func mustNewClient(c *cli.Context) client.Client {
 		if debug {
 			fmt.Fprintf(os.Stderr, "start to sync cluster using endpoints(%s)\n", strings.Join(hc.Endpoints(), ","))
 		}
-		ctx, cancel := context.WithTimeout(context.Background(), client.DefaultRequestTimeout)
+		ctx, cancel := contextWithTotalTimeout(c)
```

Comment:
Oh. I am not aware of https://github.com/coreos/etcd/blob/master/etcdctl/ctlv2/command/util.go#L302.

Then this is fine. Thanks.